wls-gui-wahllokalsystem/src/views/EreignisseView.vue

```diff
@@ -34,16 +39,27 @@ import {
 } from "vuetify/components";
 
 import BaseButtonSave from "@/components/common/buttons/BaseButtonSave.vue";
+import TheEreignisseNoEventsCheckboxes from "@/components/vorfaelleundvorkommnisse/TheEreignisseNoEventsCheckboxes.vue";
 import TheEreignisseRow from "@/components/vorfaelleundvorkommnisse/TheEreignisseRow.vue";
 import { useEreignisStore } from "@/stores/ereignisStore.ts";
 
 const ereignisStore = useEreignisStore();
+const { hasEintraege, areKeineEreignisseFlagsValid } =
+  storeToRefs(ereignisStore);
+const { addEreignis } = ereignisStore;
```

Comment:
oben im template teil kann dann bei `@click` das `ereignisStore.` weggelassen werden
```
<base-button-save
  :disabled="isSaveButtonDisabled"
  @click="sendEreignisse()"
/>
```

```suggestion
const { addEreignis, sendEreignisse } = ereignisStore;
```

Reply:
ich habe beim click-Handler eine explizite Funktion eingesetzt entsprechend der [Konvention](https://it-at-m.github.io/Wahllokalsystem/technik/naming_conventions/frontend.html#eventhandlingmethoden): d93e6cff6d247aeb8d85b02e931dcf6c0d8ed174